nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-nar-utils/src/main/java/org/apache/nifi/nar/NarClassLoaders.java

```diff
@@ -74,6 +74,7 @@ private InitContext(
             this.jettyBundle = jettyBundle;
             this.serverInstance = serverInstance;
             this.bundles = bundles;
+            System.setProperty("nifi.version", frameworkBundle.getBundleDetails().getCoordinate().getVersion());
```

Comment:
Recommend adding a comment indicating the reason for this addition. I would recommend naming this `framework.version` to avoid potential confusion that could result from different NAR extension versions, which are also part of NiFi, but could be different in particular installations.

Reply:
agreed. where are the properties documented?

Reply:
Could we name this system property `nifi.framework.version`?  This is more specific and may avoid confusion with generic system properties that are not related to NiFi.

Reply:
changed prop name to: nifi.framework.version and added a section to the Expression Language Guide.  Are there any other system properties that need to be documented?  I did not find any, outside of test classes.